Add netspeed widget

tribhuwan-kumar opened the pull request:
## Major changes
added a shell script in `src` directory for getting the real time network speed, hide the music duration from right side status bar & added `GNU bc` as requirement in `README.md`.
Tested in Arch linux, everything is working fine!

# Preview
![image](https://iili.io/JXGqy4s.png)



Maybe you should check in macOS

Enhanced #45

janoamaral reviewed (changes requested):
First and foremost, thanks for your contribution. 
I think the best approach for this is to have a setting for toggle the widgets. In this case we need 3 settings:

- @tokyo-night-tmux_show_music (1 show - 0 hide)
- @tokyo-night-tmux_show_netspeed (1 show - 0 hide)
- @tokyo-night-tmux_netspeed_iface (interface to monitor: wlan0, eth0, etc)

tribhuwan-kumar commented:
I added conditions for toggling the music widget and netspeed widget. A global variable added for getting the network interface, I removed `wlan0` from default interface and Updated the `README.md` as well!,

Thanks for this awesome theme!!

janoamaral commented on tokyo-night.tmux:
```diff
@@ -57,5 +58,5 @@ tmux set -g window-status-current-format "#[fg=#44dfaf,bg=#1F2335]   #[fg=#a9
 
 #+--- Bars RIGHT ---+
-tmux set -g status-right "$cmus_status#[fg=#a9b1d6,bg=#24283B]  %Y-%m-%d #[]❬ %H:%M $git_status$wb_git_status"
+tmux set -g status-right "$cmus_status#[fg=#a9b1d6,bg=#24283B] $netspeed $git_status$wb_git_status"
```
Restore date/time widget and remove spaces of netspeed. Also, I'm planning to move date/time to the far left. So the line should be this.


```
tmux set -g status-right "$cmus_status#[fg=#a9b1d6,bg=#24283B]$netspeed$git_status$wb_git_status#[fg=#a9b1d6,bg=#24283B]  %Y-%m-%d #[]❬ %H:%M "
```

I recommend putting netspeed after fixed length widgets because it could be anoying if the text continually jump around

janoamaral reviewed (changes requested):
I believe that with the changes I point out it should be work fine

tribhuwan-kumar commented on src/music-tmux-statusbar.sh:
```diff
@@ -77,7 +82,7 @@ if [ -n "$TITLE"  ]; then
     PLAY_STATE="$OUTPUT"
   else
-    PLAY_STATE="$OUTPUT"
+    PLAY_STATE="$OUTPUT"
```
```PLAY_STATE="$OUTPUT"```

<img src="https://github.com/janoamaral/tokyo-night-tmux/assets/118052427/a0c330c6-50f7-4758-ae7b-83f48e0960c8" width="300" height="auto" />

The original icon is removed from nerdfont, So I updated with the latest nerd font icon [ 󰏤 `f03e4`] which is same as previous icon

tribhuwan-kumar commented:
Here is the status bar after all the changes!
If any changes are necessary, please let me know 
![image_2024-03-22_11-54-37](https://github.com/janoamaral/tokyo-night-tmux/assets/118052427/756fb45c-12ea-4fde-96b2-6225934f0b81)
